contentcuration/contentcuration/viewsets/base.py

```diff
@@ -48,19 +49,58 @@ def id_attr(cls):
         info = model_meta.get_field_info(ModelClass)
         return getattr(cls.Meta, "update_lookup_field", info.pk.name)
 
-    def to_internal_value(self, data):
-        ret = super(BulkModelSerializer, self).to_internal_value(data)
+    def get_value(self, data, attr):
```

Comment:
What do these new functions do?

Reply:
They are to handle the possibility that the `key` value of a change from the frontend might be a combined index.

In this specific case, they are now an array of `[<contentnode>, <assessment_id>]` - these functions allow these id attributes to be properly handled, in the same way as if we just had a singular valued id attribute.

Reply:
Gotcha, thanks! Can you add docstrings on these new functions? So just for reference on anyone else who jumps in the future.

Reply:
Will do!